content/en/blog/news/Hertz_Open_Source_announcement/index.md

```diff
@@ -0,0 +1,66 @@
+
+## Roadmap
+For the infrastructure team, Hertz represents not only an open source project but also a genuine ultra-large-scale enterprise-level practice. By open sourcing Hertz, our aim is to enrich the Golang middleware system within the cloud-native community, enhance the CloudWeGo ecosystem, and enable more developers and enterprises to build cloud-native large-scale distributed systems. We aspire to provide modern, resource-efficient technical solutions.
+
+As mentioned earlier, Hertz has currently open sourced only the internally verified stable components. However, we intend to further enhance the framework by pursuing the following objectives:
+- **Cloud native capability support**: Integrate support for the xDS API to dynamically obtain service configurations from Istio.
+- **Multi-protocol support**: Hertz currently only open-sources HTTP1. In the future, we will open-source other protocols, such as HTTP2, Websocket, ALPN, etc., to provide developers with support for micro-service requirements in more scenarios. If you have a need, you can also submit an issue and let us know your needs for quick support.
+- **Improved command-line tools**: Continuously iterate on Hz, integrating various commonly used middleware and providing modular construction capabilities, enabling users to select the required components as needed.
+- **Enhanced ecosystem support**: Since Hertz does not adopt go net's data structure, additional ecosystem support is required. In the initial batch of open source releases, we have included CORS, Trace, and Metrics. In the future, we plan to support additional aspects such as reverse proxy and Session.
+- **Iteration based on user feedback**: Following the project's open sourcing, we will iterate based on developer feedback and requirements.
+
+We welcome everyone to [contribute](https://github.com/cloudwego/hertz/blob/develop/CONTRIBUTING.md) by submitting issues and PRs to [Hertz](https://github.com/cloudwego/hertz), fostering collaborative development. We eagerly anticipate the involvement of more developers and anticipate Hertz aiding enterprises in swiftly constructing cloud-native architectures. Enterprise users are invited to migrate and utilize Hertz, and we will provide dedicated technical support and communication. Feel free to [join our slack community](https://join.slack.com/t/cloudwego/shared_invite/zt-tmcbzewn-UjXMF3ZQsPhl7W3tEDZboA) for further consultations and assistance.
+
+## Related Links
+- Project [address](https://github.com/cloudwego/hertz)
+- Community [ecosystem](https://github.com/hertz-contrib)
+- Hertz benchmark [blog](https://github.com/cloudwego/cloudwego.github.io/tree/main/content/en/blog/news/Hertz_Benchmark)
```

Comment:
Please replace with this link: https://www.cloudwego.io/blog/2023/02/24/getting-started-with-hertz-performance-testing-guide/